### PR TITLE
Revert custom CIDR change

### PR DESCRIPTION
**Pull request by colemickens**
Reverts #473.

Sending for review/approval. Still seeing if I can avoid backing this out.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/508)

### CODE REVIEW

File: parts/kubernetesmastervars.t [hunk 1]
```diff
@@ -179,7 +178,7 @@
     "agentWindowsSku": "2016-Datacenter-with-Containers",
     "agentWindowsVersion": "2016.0.20170127",
     "singleQuote": "'",
-    "windowsCustomScriptSuffix": " $inputFile = '%SYSTEMDRIVE%\\AzureData\\CustomData.bin' ; $outputFile = '%SYSTEMDRIVE%\\AzureData\\CustomDataSetupScript.ps1' ; Copy-Item $inputFile $outputFile ; Start-Job -command { Invoke-Expression('{0} {1}' -f $outputFile, $arguments) } ; "
```
Comment (JackQuincy) [inline parts/kubernetesmastervars.t]: was this part of the custom cidr? I  thought you added these changes seperately

File: parts/kuberneteswinagentresourcesvmas.t [hunk 2]
```diff
@@ -156,7 +156,7 @@
         "typeHandlerVersion": "1.8",
         "autoUpgradeMinorVersion": true,
         "protectedSettings": {
-          "commandToExecute": "[concat('start /b powershell.exe -ExecutionPolicy Unrestricted -command \"', '$arguments = ', variables('singleQuote'),'-MasterIP ',variables('masterInternalLbIp'),' -KubeDnsServiceIp ',variables('kubeDnsServiceIp'),' -MasterFQDNPrefix ',variables('masterFqdnPrefix'),' -Location ',variables('location'),' -AgentKey ',variables('clientPrivateKey'),' -AzureHostname ',variables('{{.Name}}VMNamePrefix'),copyIndex(variables('{{.Name}}Offset')),' -AADClientId ',variables('servicePrincipalClientId'),' -AADClientSecret ',variables('servicePrincipalClientSecret'),variables('singleQuote'), ' ; ', variables('windowsCustomScriptSuffix'), '\" > %SYSTEMDRIVE%\\AzureData\\CustomDataSetupScript.log 2>&1')]"
```
Comment (JackQuincy) [inline parts/kuberneteswinagentresourcesvmas.t]: this also feel unrelated to custom cidr

File: parts/kubernetesmastervars.t (hunk 1 above)
Comment (colemickens) [inline parts/kubernetesmastervars.t]: intentional. see #501

File: parts/kuberneteswinagentresourcesvmas.t (hunk 2 above)
Comment (colemickens) [inline parts/kuberneteswinagentresourcesvmas.t]: intentional. see #501

File: parts/kubernetesmastervars.t (hunk 1 above)
Comment (JackQuincy) [inline parts/kubernetesmastervars.t]: so you are intentionaly removing these changes? that 501 introduced?

Comment (colemickens) [inline parts/kubernetesmastervars.t]: 501 is removing the Start-Job: https://github.com/Azure/acs-engine/pull/501/files#diff-f18f17bd6f84d54a111be39c8fe59d54R182

I guess I don't even understand why this diff is showing up here.

Comment (colemickens) [inline parts/kubernetesmastervars.t]: I think I screwed up the base for this commit and then didn't force push after I fixed it. The commit was anthony's commit, which is why the diff looked really weird (and probably why it wasn't working). I just force pushed, let's see if it looks better.

Comment (colemickens) [inline parts/kubernetesmastervars.t]: There we go, this looks better. Thanks for catching it.

**Comment by JackQuincy** [comment]
@acs-bot test this please

**Comment by asktomsk** [comment]
Could anyone point why #473 was backed out please?

**Comment by JackQuincy** [comment]
It broke our CNI plugin. (pod to pod communication). We have work planned to get them working together. It doesn't make sense to any of us why they are incompatible. Any one who wants to help investigate why they don't work together we would definitely appreciate.
